locales/en-US/snoozetabs.properties

```diff
@@ -1,35 +1,45 @@
-# LOCALIZATION NOTE: This add-on will let you select tabs to close for a while, after which they will re-appear in your browser with a notification.
+# This add-on will let you select tabs to close for a while, after which they
```

Comment:
Hmm, do we not need `LOCALIZATION NOTE:` at the start of comments to have them show up in Pontoon?